### PR TITLE
added validations to user models

### PR DESCRIPTION
**Pull request by MKaiKai**
Please note the validations for email and booking status are pretty generic at the moment; they may change depending on how we choose to code them

Also, please check my usage of numericality for the integer specific attributes(?) i.e fee, capacity, lesson_length_minutes

Thaaaanks!

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class Booking < ApplicationRecord
   belongs_to :user
   belongs_to :lesson
+
+  validates :start_date, presence: true
+  validates :status, presencef: true
```
Comment (mcosme000) [inline app/models/booking.rb]: ⚠️ typo! please correct to `presence: true` 🙏🏻

File: app/models/lesson.rb [hunk 2]
```diff
@@ -2,5 +2,11 @@ class Lesson < ApplicationRecord
   has_many :bookings
   belongs_to :user
 
-  validate :users, :email, :address, :description, :capacity, :cuisine_genre, :fee, :lesson_length_minutes, presence: true
+  validates :name, length: {minimum: 3}
+  validates :address, presence: true
+  validates :cuisine_genre, presence: true
+  validates :description, length: {in: 10..75}
```
Comment (mcosme000) [inline app/models/lesson.rb]: I think we can allow longer descriptions, what about up to 200 characters?

File: app/models/user.rb [hunk 3]
```diff
@@ -4,6 +4,8 @@ class User < ApplicationRecord
   has_many :lessons
   has_many :bookings
 
+  validates :email, presence: true
```
Comment (mcosme000) [inline app/models/user.rb]: For now is okay, but we can also validate with a regex!

**Review by mcosme000 — COMMENTED** [review]
Thank you M! Please correct the typo en the Booking model and maybe increase the character length for description in the Lesson model

File: app/models/booking.rb (hunk 1 above)
Comment (MKaiKai) [reply app/models/booking.rb]: whoopsie! thaaaanks!

File: app/models/lesson.rb (hunk 2 above)
Comment (MKaiKai) [reply app/models/lesson.rb]: done!

File: app/models/user.rb (hunk 3 above)
Comment (MKaiKai) [reply app/models/user.rb]: ahhhh! yes regex

will this work?

 validates :email,  format: { with: /\A[^@\s]+@([^@.\s]+\.)+[^@.\s]+\z/ }

**Comment by MKaiKai** [comment]
- The typo in the booking model has been fixed
- The description length in the lesson model has been increased to 200
- The email validation in the user model has been updated to include regex

**Pull request merged**